scripts/tests/provisioning/test.expect

```diff
@@ -47,7 +47,7 @@ expect {
 		send "opkg install init-restore-mode-ptest\r"
 	}
 	timeout {
-		send_user "\nTIMEOUT waiting for opkg to configure and ptests to install.\n"
+		send_user "\nTIMEOUT waiting for shell.\n"
```

Comment:
And do you need a 90s timeout for the shell to appear?

Reply:
You shouldn't *need* one. But that's how I have it written, yes.